testing/go/framework.go

```diff
@@ -167,8 +170,13 @@ func RunScripts(t *testing.T, scripts []ScriptTest) {
 	if len(focusScripts) > 0 {
 		scripts = focusScripts
 	}
+	
 	for _, script := range scripts {
-		RunScript(t, script)
+		if runOnPostgres {
+			RunScriptOnPostgres(t, script)
```

Comment:
I'd actually use the variable to make the distinction inside of `RunScript`. Seems weird to me that setting the variable makes a difference in `RunScripts` but not `RunScript`.

Reply:
Done